resources/js/Components/Form/DmEntryCreateForm.tsx

```diff
@@ -0,0 +1,479 @@
+                                    Cancel
+                                </Button>
+                            )}
+                        </Grid>
+                        <Grid item md={type === 'Edit' ? 4 : 8} />
+                        <Grid item md={type === 'Edit' ? 4 : 2} xs={6}>
+                            <Button
+                                variant='contained'
+                                onClick={() => setActiveStep(1)}
+                                fullWidth>
+                                Continue
+                            </Button>
+                        </Grid>
+                    </>
+                )}
+                {activeStep === 1 && (
+                    <>
+                        <Grid container spacing={2}>
+                            <Grid item md={type === 'Edit' ? 4 : 2} xs={4}>
+                                <Button
+                                    onClick={() => {
+                                        setActiveStep(0)
+                                        setIsItemsVisible(false)
+                                    }}
+                                    fullWidth>
+                                    Previous
+                                </Button>
+                            </Grid>
+                            <Grid item md={type === 'Edit' ? 4 : 6} />
+                            <Grid item md={type === 'Edit' ? 4 : 2} xs={4}>
+                                {type === 'Create' ? (
+                                    <Button
+                                        variant='contained'
+                                        fullWidth
+                                        onClick={() => {
+                                            if (
+                                                !(
+                                                    items.filter(
+                                                        (e) => e.name === item_data.name,
+                                                    ).length > 0
+                                                )
+                                            ) {
+                                                setError(false)
+                                                setItems([...items, item_data])
+                                                setData('items', [...items, item_data])
+                                                setIsItemsVisible(true)
+                                                setItemData(
+                                                    ITEM_CREATE_FORM_INITIAL_VALUE,
+                                                )
+                                            } else {
+                                                setError(true)
+                                            }
+                                        }}>
+                                        Add Item
+                                    </Button>
+                                ) : null}
```

Comment:
don't use ternary operator when the other condition is null, rahter do `type === 'Create' && ()`